prep-javascript-variables/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      /* your code here */
+      const fullName = 'Duncan Harrison';
+      console.log(fullName);
+      console.log(typeof 'Duncan Harrison');
```

Comment:
This should be `typeof fullName` instead. You want to check what the type of your variable is